src/UiElement/UiElementTrait.php

```diff
@@ -64,6 +64,14 @@ public function getAlias(): ?string
         return $this->metadata->getAlias();
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function isEnabled(): bool
```

Comment:
Where is this method used?
Is its purpose to be used inside the template? It could since we have access to this object in the template.